docs/getting-started.md

```diff
@@ -58,7 +58,32 @@ const {ChromaClient} = require('chromadb');
 const client = new ChromaClient();
 ```
 
-To connect to Chroma's backend - you either need to connect to a hosted version of Chroma, or run it on your local computer. If you can run `docker-compose up -d --build` you can run Chroma. 
+To connect to Chroma's backend - you either need to connect to a hosted version of Chroma, or run it on your local computer. If you can run `docker-compose up -d --build` you can run Chroma.
+
+:::caution Ephemeral Storage 💀
```

Comment:
can we simplify this whole section? I think either it should fit in one small callout, or we should find a different place to put it other than `getting started`

Reply:
We can add it to the Troubleshooting section and just link to it.

```
:::caution Ephemeral Storage 💀
By default docker compose will store your data in a local volume see <link to troubleshooting>
:::
```

Or maybe we add a new Tips&Tricks/FAQ section where to put such tidbits, wdyt?